LoRaEngine/modules/LoRaBasicsStationModule/station.conf

```diff
@@ -20,6 +20,8 @@
         /* chan_multiSF_X, chan_Lora_std, chan_FSK provided by LNS */
     },
     "station_conf": {
+        "placeholder": "placeholder",
+        "logfile_size": "10mb",
```

Comment:
was this new addition on purpose?

Reply:
Nope will remove thanks for the the catch